File: cleanup.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, 'src/pages/Admin.jsx');

try {
  let content = fs.readFileSync(filePath, 'utf-8');
  const lines = content.split('\n');
  
  // Trouver la première occurrence de 'export default Admin'
  let exportLine = -1;
  for (let i = 0; i < lines.length; i++) {
    if (lines[i].includes('export default Admin')) {
      exportLine = i;
      break;
    }
  }
  
  if (exportLine !== -1) {
    // Garder juste jusqu'à exportLine (include la ligne export)
    const cleanLines = lines.slice(0, exportLine + 1);
    const cleanContent = cleanLines.join('\n');
    
    fs.writeFileSync(filePath, cleanContent, 'utf-8');
    console.log(`Fichier nettoyé! Gardé ${exportLine + 1} lignes, supprimé ${lines.length - exportLine - 1} lignes dupliquées`);
  } else {
    console.log('export default Admin not found!');
  }
} catch (error) {
  console.error('Erreur:', error.message);
}

File: cleanup.py
import os

file_path = 'src/pages/Admin.jsx'

with open(file_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# Trouver la première occurrence de 'export default Admin'
export_line = -1
for i, line in enumerate(lines):
    if 'export default Admin' in line:
        export_line = i
        break

if export_line != -1:
    # Garder juste jusqu'à export_line+1 (include la ligne export)
    clean_lines = lines[:export_line+1]
    with open(file_path, 'w', encoding='utf-8') as f:
        f.writelines(clean_lines)
    print(f"Fichier nettoyé! Gardé {export_line+1} lignes, supprimé {len(lines)-export_line-1} lignes dupliquées")
else:
    print("export default Admin not found!")
